lib/report.js

```diff
@@ -104,23 +104,26 @@ class ConciseReporter extends Reporter {
     ['type', 'message', 'expected', 'actual', 'stack']
       .filter(key => res.hasOwnProperty(key))
       .forEach(key => {
-        let value = res[key];
```

Comment:
With these changes you don't use `value` below except for `value instanceof Error` twice; thus I think it'd be better just to extract the value of `value instanceof Error` into the variable (called something like `isError`) and avoid making other changes (such as adding the `valueStr` variable).